v-next/hardhat/src/internal/builtin-global-options.ts

```diff
@@ -58,6 +58,17 @@ export const BUILTIN_GLOBAL_OPTIONS_DEFINITIONS: GlobalOptionDefinitions =
         }),
       },
     ],
+    [
+      "verbosity",
+      {
+        pluginId: "builtin",
+        option: globalLevel({
+          name: "verbosity",
```

Comment:
Sorry I didn't notice this before, but I think having both `verbose` and `verbosity` options might be confusing. What do you think of using a more (ahem) verbose name like `--trace-verbosity`, `--execution-trace-verbosity`, `--test-verbosity`, or something like that?

Reply:
I created a thread for this on Slack so we could discuss the options in one place - https://nomicfoundation.slack.com/archives/C03P6B72ZHU/p1752064116423929